build.rs

```diff
@@ -3,34 +3,67 @@ extern crate serde;
 extern crate serde_json;
 
 use serde_json::Value;
+use std::env;
+use std::ffi::OsString;
 use std::fs::{self, File};
 use std::io::{BufReader, Write};
 
-#[derive(Deserialize, Debug)]
+#[derive(Serialize, Deserialize, Debug)]
 struct TestData {
     avro: Value,
     bigquery: Value,
     json: Value,
 }
 
-#[derive(Deserialize, Debug)]
+#[derive(Serialize, Deserialize, Debug)]
 struct TestCase {
+    #[serde(default, skip_serializing_if = "Value::is_null")]
+    description: Value,
     name: String,
     test: TestData,
 }
 
-#[derive(Deserialize, Debug)]
+#[derive(Serialize, Deserialize, Debug)]
 struct TestSuite {
     name: String,
     tests: Vec<TestCase>,
```

Comment:
Ditto the description field here too since the test suite has a top-level description.